wandb/wandb_run.py

```diff
@@ -390,6 +391,23 @@ def save(self, id=None, program=None, summary_metrics=None, num_retries=None, ap
         self.name = upsert_result.get('displayName')
         return upsert_result
 
+    def check_anonymous(self):
+        # If there's no API key set, ask if the run should be logged anonymously. Only launch this prompt in
+        # environments with a tty.
+        if not self.api.api_key and sys.stdin.isatty():
+            resp = str(input('No API key found. Would you like to log runs anonymously? (y/n)\n').lower().strip())
```

Comment:
We need to make this clearer it's wandb asking for input.  Ideally we would prefix it with the blue `wandb:` and print the app_url in this message, i.e. log runs anonymously to https://app.wandb.ai

Reply:
![Screen Shot 2019-07-30 at 3 29 19 PM](https://user-images.githubusercontent.com/1299990/62170037-dbb3fa80-b2de-11e9-97c9-5ebbb6e75c6c.png)

How does this look?

Reply:
Looks good, one minor nit would be to make the Appending key message have wandb: as well, but that's been like that forever.